ansible_collections/infoblox/b1ddi_modules/plugins/modules/b1_ns_record_gather.py

```diff
@@ -77,6 +77,10 @@ def get_ns_record_gather(data):
     flag=0
     fields=data['fields']
     filters=data['filters']
+    if 'name' in filters:
+        filters['dns_name_in_zone'] = filters.pop('name')
+    if 'nameserver' in filters:
```

Comment:
What does the nameserver denotes, is it the dname?

Reply:
Yes, in NIOS we use "nameserver" and "canonical" for "name" and "cname" respectively. Will change both the fields according to "rdata".